### PR TITLE
Add popup window and js code

### PR DESCRIPTION
**Pull request by NabilHY**
In this branch, I have implemented a popup window that includes the project details on both desktop and mobile.

- Using a JavaScript array to store all of the information for all projects.
- While sicking to the Figma template

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -10,6 +10,7 @@ html {
```
Comment (elyor-doniyorov) [inline styles.css:10]: - [ ] Make sure the popup follows the design for both desktop and mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, tests, margins). Please take a look at the below screenshots to find the differences and work on them 👇 
- Figma desktop popup 👇 
Screenshot (216)
- Your desktop popup 👇 
Screenshot (212)
- Figma mobile popup 👇 
Screenshot (214)
- Your mobile popup 👇 
Screenshot (213)

File: index.html [hunk 2]
```diff
@@ -1,380 +1,293 @@
...
+            alt="Access to GitHub Account" /></a>
+      </li>
+      <li>
+        <a href="https://angel.co/"><img class="icons" src="images/angellist.png"
+            alt="Access to Angellist Account" /></a>
+      </li>
+    </ul>
+  </section>
+
+  <section id="portfolio" class="works">
```
Comment (elyor-doniyorov) [inline index.html]: - [ ] Also make sure the projects section should be created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed, and instead it should be added via JavaScript.** Check an example of students projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @NabilHY, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (NabilHY) [reply index.html]: - Thank you for reviewing my code, I made the changes that you suggested, which include : 
- Making the work section gets created dynamically through JS implementation.
- I edited the styling as much as I could to match the one on the Figma template.

File: styles.css (hunk 1 above)
Comment (Meltrust) [inline styles.css:10]: - [ ] Nice work, but there is still a little more to do about the fonts.

Everything is important:  size, weight, line-height, font family, style.   It seems you have been misregarding the `font-weight` and the `line-height` properties for the fonts.  Please include them so the project looks like the figma.

- [ ] There is a problem with the languages boxes (HTML, CSS, javascript, etc).  Here is the solution:

image

Just remove that `width: 20%` property and it should work! (tested of firefox and chrome)💪 

- [ ] These commas are not featured on the Figma, therefore, please kindly remove them:

image

If you implement these changes, your project will look like Figma! 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @NabilHY,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

You can do it

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (NabilHY) [reply styles.css:10]: Thanks for addressing those issues, and I appreciate your feedback it was very clear.
I've made the following changes :
- Removed the width property from the list items.
- Removed commas by using the JS join method

**Review by thecodechaser — APPROVED** [review]
## Hi @NabilHY 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## optional suggestions
- [ ]  Please include a description of your pull request.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**